server/backup_runner.go

```diff
@@ -14,126 +15,180 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	backupExpireDuration  = 7 * 24 * time.Hour
+	backupCleanupInterval = 30 * time.Minute
```

Comment:
nit: can we do it together with backup? Clean up the backup that should be archived after the backup is done successfully.

Also, how does the customer configure backupExpireDuration?